source/_integrations/opentherm_gw.markdown

```diff
@@ -254,3 +255,11 @@ Possible LED modes and their meaning are listed here:
 - E. Transmission error has been detected.
 - M. Boiler requires maintenance.
 - P. Raised power mode active on thermostat interface.
+
+## Advanced usage
```

Comment:
```suggestion
## Switches
```
Would it make sense to rename this section to switches, in case someone looks for a description of the switches?

Reply:
There will be different entity types added in the near future with the same considerations. Maybe something like `Disabled configuration entities` would be more appropriate?

Reply:
In that case, yes.